extensions/pkg/controller/backupentry/genericactuator/actuator.go

```diff
@@ -132,6 +132,10 @@ func (a *actuator) deployEtcdBackupSecret(ctx context.Context, log logr.Logger,
 			if token, ok := etcdSecret.Data[securityv1alpha1constants.DataKeyToken]; ok {
 				etcdSecretData[securityv1alpha1constants.DataKeyToken] = token
 			}
+		} else {
+			// Unset all annotations and labels when static credentials are used.
+			etcdSecret.Annotations = map[string]string{}
```

Comment:
Nit: this I think is more consistent with the assignment of the labels
```suggestion
			etcdSecret.Annotations = nil
```

Reply:
The reason I did not set it to `nil` is that the actuator always sets at least one annotation key-value pair, i.e. a map needs to be allocated. 

I see `metav1.SetMetaDataAnnotation` allocates a map if the annotation is `nil`, but I see nothing wrong if we do it instead of 3rd party function.